game/src/js/scenes/TitleScene.js

```diff
@@ -68,4 +89,5 @@ export default class TitleScene extends Phaser.Scene {
       this.scene.start('DungeonScene');
```

Comment:
We can now also get rid of these key events, since we now start the game with the buttons.